3rdparty/openvx/include/openvx_hal.hpp

```diff
@@ -0,0 +1,227 @@
+#ifndef OPENVX_HAL_HPP_INCLUDED
+#define OPENVX_HAL_HPP_INCLUDED
+
+#include <cstdio>
+#include "opencv2/core/hal/interface.h"
+#include "VX/vx.h"
+#include "VX/vxu.h"
+#include <string>
+#include <iostream>
+
+//===================================
+// utility
+// ...
+
+#include <chrono>
```

Comment:
agree; if I remember correctly, we have TickMeter class or something like that in OpenCV - let's use it